docs/topics/db/transactions.txt

```diff
@@ -290,45 +290,47 @@ Performing actions after commit
 
 Sometimes you need to perform an action related to the current database
 transaction, but only if the transaction successfully commits. Examples might
-include a `Celery`_ task, an email notification, or a cache invalidation.
+include a background task, an email notification, or a cache invalidation.
 
-.. _Celery: https://pypi.org/project/celery/
-
-Django provides the :func:`on_commit` function to register callback functions
-that should be executed after a transaction is successfully committed:
+:func:`on_commit` allows you to register callbacks that will be executed after
+the open transaction is successfully committed:
 
 .. function:: on_commit(func, using=None, robust=False)
 
-Pass any function (that takes no arguments) to :func:`on_commit`::
+Pass a function, or any callable, to :func:`on_commit`::
 
     from django.db import transaction
 
-    def do_something():
-        pass  # send a mail, invalidate a cache, fire off a Celery task, etc.
+    def send_welcome_email():
+        ...
 
-    transaction.on_commit(do_something)
+    transaction.on_commit(send_welcome_email)
 
-You can also bind arguments to your function using :func:`functools.partial`::
+Callbacks will not be passed any arguments, but you can bind them with
+:func:`functools.partial`::
```

Comment:
moved argument-free spec here, to where we describe why to use args

Reply:
restore `bind them` -> `bind arguments` maybe? (or something like it)

Rationale: "them" _could_ be read as applying to either "arguments" or "callbacks" (though the sentence parses as correct enough English [AFAIK 😄], that's a reasonably nuanced bar)

Reply:
I think it's reasonable to assume the reader will understand that it's binding the arguments. If they are in doubt, I think the code example and `partial()` docs will clarify.